packages/synapse-interface/constants/tokens/index.ts

```diff
@@ -135,6 +135,8 @@ export const tokenAddressToToken = (
   if (chainId) {
     if (tokenAddress === WETH.addresses[chainId]) {
       return WETH
+    } else if (tokenAddress === '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE') {
```

Comment:
'constants/index.ts' file has this address so dont have to hardcode here

Reply:
missed that - thank you! updated in https://github.com/synapsecns/sanguine/pull/2923/commits/bdeabe54348e7559c744b4dd87affbb3d2724d3e